administering-tugboat-crew/index.md

```diff
@@ -0,0 +1,129 @@
+# Administering your Tugboat crew
+
+Every ship captain needs a crew - even a tugboat! Want to add, remove or make
+changes to your Tugboat team?
+
+- [Add a user to a project](#add-a-user-to-a-project)
+- [Remove a user from a project](#remove-a-user-from-a-project)
+- [Change user permissions](#change-user-permissions)
+- [Add a Tugboat bot to your team](#add-a-tugboat-bot-to-your-team)
+- [User permission levels explained](#user-permission-levels-explained)
+
+> #### Info::Permissions and organizing projects
+>
+> User permissions in Tugboat are handled on a project basis - not based on the
+> repository. If you've got discrete sets of users who should not be able to
+> access a specific repo; for example, you've got two repos, one containing an
+> internal 'top secret' project, and one containing a front-facing static site
+> that an external vendor manages; you'll want to make those different projects.
+
+## Add a user to a project
+
+1. Click your username in the upper right-hand corner, and go to Projects;
+2. Click the **My Projects** link;
```

Comment:
I think we can consolidate steps 1 & 2 by providing a link directly to 'My Projects'. https://dashboard.tugboat.qa/projects

Reply:
I used different phrasing in some of the other docs that consolidated these steps; "Go to username -> My Projects at the upper-right of the Tugboat screen." I'll consolidate those steps everywhere I've listed them out separately, and make 'My Projects' a clickable link in that step.